src/ChatGptNet/ChatGptClient.cs

```diff
@@ -113,38 +113,44 @@ public async IAsyncEnumerable<ChatGptResponse> AskStreamAsync(Guid conversationI
         {
             var contentBuilder = new StringBuilder();
 
-            using (var responseStream = await httpResponse.Content.ReadAsStreamAsync(cancellationToken))
+            await using (var responseStream = await httpResponse.Content.ReadAsStreamAsync(cancellationToken))
```

Comment:
Why did you change the `AskStreamAsync` method? It is not related the the issue #55.